src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Ball ball1 = new Lottery().getRandomBall();
+        System.out.println(ball1);
+
+        Ball ball2 = new Lottery().getRandomBall();
+        System.out.println(ball2);
+
+        Ball ball3 = new Lottery().getRandomBall();
```

Comment:
It's not efficient to create a new instance of the Lottery class each time you want to get a random ball. Consider creating a single instance of the Lottery class and using it to get all random balls.